project/src/test/java/com/google/sps/SearchStoreTest.java

```diff
@@ -38,66 +41,156 @@
+  private static final String NAME_WITHOUT_FOOD_VENDORS = "Wednesday Cesar Chavez Farmers' Market";
+  private static final ImmutableList<Keyword> KEYWORDS_NAME_WITHOUT_FOOD_VENDORS =
+      ImmutableList.of(new Keyword("farmers' market", 0.54f), new Keyword("cesar chavez", 0.26f));
+  private static final String DESCRIPTION_WITH_FOOD_VENDORS =
+      "Weekly farmers' market with certified growers and hot food vendors."
+          + "Event hours are 10am to 1:30pm.";
+  private static final ImmutableList<Keyword> KEYWORDS_DESCRIPTION_WITH_FOOD_VENDORS =
+      ImmutableList.of(
+          new Keyword("farmers' market", 0.36f),
+          new Keyword("food vendors", 0.26f),
+          new Keyword("growers", 0.26f),
+          new Keyword("Event", 0.12f));
+  private static final String FOOD = "food";
+  private static final ImmutableMap<String, String> TOKENS_NAME_WITHOUT_FOOD_VENDORS =
+      ImmutableMap.<String, String>builder()
+          .put("wednesday", "wednesday")
+          .put("cesar", "cesar")
+          .put("chavez", "chavez")
+          .put("farmers", "farmer")
+          .put("'", "'")
+          .put("market", "market")
+          .build();
+  private static final ImmutableMap<String, String> TOKENS_DESCRIPTION_WITH_FOOD_VENDORS =
+      ImmutableMap.<String, String>builder()
+          .put("Weekly", "Weekly")
+          .put("farmers", "farmer")
+          .put("'", "'")
+          .put("market", "market")
+          .put("with", "with")
+          .put("certified", "certified")
+          .put("growers", "grower")
+          .put("and", "and")
+          .put("hot", "hot")
+          .put("food", "food")
+          .put("vendors", "vendor")
+          .put(".", ".")
+          .build();
+  private static final String VENDOR = "vendor";
+  private static final String NAME_WITHOUT_GROWERS = "Wednesday Cesar Chavez Farmers' Market";
+  private static final ImmutableList<Keyword> KEYWORDS_NAME_WITHOUT_GROWERS =
+      ImmutableList.of(
+          new Keyword("farmers' market", 0.54f),
+          new Keyword("cesar chavez", 0.26f));
+  private static final String DESCRIPTION_WITH_GROWERS =
+      "Weekly farmers' market with certified growers and hot food vendors."
+      + "Event hours are 10am to 1:30pm.";
+  private static final ImmutableList<Keyword> KEYWORDS_DESCRIPTION_WITH_GROWERS =
+      ImmutableList.of(
+          new Keyword("farmers' market", 0.36f),
+          new Keyword("food vendors", 0.26f),
+          new Keyword("growers", 0.26f),
+          new Keyword("Event", 0.12f));
+  private static final ImmutableMap<String, String> TOKENS_NAME_WITHOUT_GROWERS =
+      ImmutableMap.<String, String>builder()
+          .put("wednesday", "wednesday")
+          .put("cesar", "cesar")
+          .put("chavez", "chavez")
+          .put("farmers", "farmer")
+          .put("'", "'")
+          .put("market", "market")
+          .build();
+  private static final ImmutableMap<String, String> TOKENS_DESCRIPTION_WITH_GROWERS =
```

Comment:
same as TOKENS_DESCRIPTION_WITH_FOOD_VENDORS ?

Reply:
Done. Used putAll from ImmutableMap to create a copy of first map.